src/hermes/commands/deposit/invenio.py

```diff
@@ -12,10 +12,15 @@
 
 import click
 
+from hermes import config
 from hermes.commands.deposit.error import DepositionUnauthorizedError
 from hermes.model.context import CodeMetaContext
 from hermes.model.path import ContextPath
 
+_DEFAULT_SITE_URL = "https://sandbox.zenodo.org"
```

Comment:
Is it really the "sandbox" that we want to default to?

Reply:
Yes, that sounds good. However, as long as we're not released, it's okay to faciliate testing... or we need to also start having example configs / test configs.

To sum up: This change would not block an approval from my side ;)

Reply:
I'm throwing an exception now in 2bd83ad35029840e06950d388c9628c577d5754c